libs/community/tests/unit_tests/vectorstores/test_databricks_vector_search.py

```diff
@@ -681,3 +681,32 @@ def test_similarity_search_by_vector_not_supported_for_managed_embedding() -> No
         "`similarity_search_by_vector` is not supported for index with "
         "Databricks-managed embeddings." in str(ex.value)
     )
+
+
+@pytest.mark.requires("databricks", "databricks.vector_search")
+@pytest.mark.parametrize(
+    "method",
+    [
+        "similarity_search",
+        "similarity_search_with_score",
+        "similarity_search_by_vector",
+        "similarity_search_by_vector_with_score",
+        "max_marginal_relevance_search",
+        "max_marginal_relevance_search_by_vector",
+    ],
+)
+def test_filter_arg_alias(method: str) -> None:
```

Comment:
Since we're greedily using `filter` and only popping `filters` from `kwargs` if `filter` is not set, can we add a validation of behavior if both `filter` and `filters` are specified?

Reply:
The ``kwargs`` should not contain `filter` because they are explicitly named in the args of caller functions. If both `filter` and `filters` are specified in the caller e.g. `similarity_search(filter=xxx, filters=yyy)`, the `filter` will precede. I think I can add that test case.